### PR TITLE
MWPW-141392: Tooltip On Focus & Aria-checked

### PR DESCRIPTION
**Pull request by joaquinrivero**
Adding isChecked prop for radio button states, adjusting aria-checked attributes, and handling tooltips for screen reader support

Resolves: [MWPW-141392](https://jira.corp.adobe.com/browse/MWPW-141392)
Resolves: [MWPW-141394](https://jira.corp.adobe.com/browse/MWPW-141394)
Resolves: [MWPW-141395](https://jira.corp.adobe.com/browse/MWPW-141395)

Test URLs:
Before: https://main--dc--adobecom.hlx.live/acrobat/online/compress-pdf?martech=off
After: https://main--dc--adobecom.hlx.live/acrobat/online/compress-pdf?milolibs=mwpw-141392--milo--joaquinrivero&martech=off

URL for testing:
- https://mwpw-141392--milo--joaquinrivero.hlx.page/

### CODE REVIEW

File: libs/blocks/review/components/review/Ratings.js [hunk 1]
```diff
@@ -125,6 +126,7 @@ const Ratings = ({
       ref=${fieldSetRef}
       className="hlx-Review-ratingFields"
       onFocus=${onFocus}
+      onKeyUp=${onFocus}
```
Comment (overmyheadandbody) [inline libs/blocks/review/components/review/Ratings.js:129]: Should the tooltip not be removed after focus exits the area?

**Review by seanchoi-dev — DISMISSED** [review]
Please fix the eslint issue and review the unit test.

File: libs/blocks/review/components/review/RatingInput.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ const RatingInput = ({
     <input
       data-tooltip=${tooltip}
```
Comment (Axelcureno) [inline libs/blocks/review/components/review/RatingInput.js:52]:
```suggestion
      data-tooltip="${tooltip}"
```

Comment (Axelcureno) [reply libs/blocks/review/components/review/RatingInput.js:52]: Same in other attrs below.. double quotes are missing. Is this expected?

**Comment by Axelcureno** [comment]
`aem-psi-check` requires a test URL from the forked repo. It won't recognize the URL if it's *.adobecom.hlx.* and your branch is from your fork. I've already updated it in your ticket description so just FYI for next time.

**Review by Blainegunn — APPROVED** [review]
lgtm, thank you @joaquinrivero

**Review by Axelcureno — APPROVED** [review]

File: libs/blocks/review/components/review/Ratings.js [hunk 3]
```diff
@@ -87,7 +90,7 @@ const Ratings = ({
   };
 
   const onBlur = (ev) => {
-    if (ev.relatedTarget === null || ev.relatedTarget.nodeName !== 'INPUT') {
+    if (ev.relatedTarget === null || ev?.relatedTarget?.nodeName !== 'INPUT') {
```
Comment (nkthakur48) [inline libs/blocks/review/components/review/Ratings.js:93]: Looks like you are checking presence of `ev` (`ev?.relatedTarget?.nodeName`). But if `ev` is undefined, then you will get an exception while doing `ev.relatedTarget`

**Review by nkthakur48 — COMMENTED** [review]
Minor comment!

**Review by nkthakur48 — APPROVED** [review]

**Pull request merged**